fastly/http3.go

```diff
@@ -0,0 +1,115 @@
+package fastly
+
+import (
+	"fmt"
+	"time"
+)
+
+// HTTP3 represents a response from the Fastly API.
+type HTTP3 struct {
+	CreatedAt       *time.Time `mapstructure:"created_at"`
+	DeletedAt       *time.Time `mapstructure:"deleted_at"`
+	FeatureRevision int        `mapstructure:"feature_revision"`
+	ServiceID       string     `mapstructure:"service_id"`
+	ServiceVersion  int        `mapstructure:"version"`
+	UpdatedAt       *time.Time `mapstructure:"updated_at"`
+}
```

Comment:
It may be useful to add corresponding `json` field tags so that `json.Marshal(HTTP3)` works as expected.

Reply:
Thanks @awilliams-fastly yup I saw you doing this recently and that sounds like a good general practice 👍🏻